app/cafe.py

```diff
@@ -0,0 +1,19 @@
+from app.errors import NotVaccinatedError, \
+                        OutdatedVaccineError, \
+                        NotWearingMaskError
+import datetime
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, vstr: dict) -> str:
```

Comment:
- variable name should be more specific, the current name indicates that the variable is a string.
- for better typing you can use typing.Dict which let's you specify the type of the keys and values

```suggestion
    from typing import Dict

    def visit_cafe(self, visitor_data: Dict[str, Any]) -> str:
```

ref. [typing.Dict vs dict](https://stackoverflow.com/questions/37087457/difference-between-defining-typing-dict-and-dict)